PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Support Files/Info.plist

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSAppTransportSecurity</key>
```

Comment:
You shouldn't need this option. Any reason about why you added it?

Reply:
Some assets don't play without it. I saw it in some ooyala wiki.